src/asm/rgbasm.1

```diff
@@ -105,8 +203,11 @@ $ rgbasm -o bar.o - < foo.asm
 The resulting object file is not yet a usable ROM image \(em it must first be
 run through
 .Xr rgblink 1
-and
+and then
 .Xr rgbfix 1 .
+.Sh BUGS
+Please report bugs on
+.Lk https://github.com/rednex/rgbds/issues GitHub
```

Comment:
End in a period:

    .Lk https://github.com/rednex/rgbds/issues GitHub .

Reply:
Will do!